src/com/clover/remote/client/transport/websocket/WebSocketPairedCloverTransport.ts

```diff
@@ -1,21 +1,9 @@
-import sdk = require('remote-pay-cloud-api');
-import http = require('http');
-
-import {RemoteMessageParser} from '../../../../json/RemoteMessageParser';
+import * as sdk from 'remote-pay-cloud-api';
 
 import {PairingDeviceConfiguration} from '../PairingDeviceConfiguration';
-import {CloverDeviceConfiguration} from '../../device/CloverDeviceConfiguration';
-import {CloverDevice} from '../../device/CloverDevice';
 import {CloverWebSocketClient} from './CloverWebSocketClient';
-
-import {CloverTransport} from '../CloverTransport';
-import {Logger} from '../../util/Logger';
```

Comment:
Logger is being used, need to keep this...

Reply:
Logger is imported in the parent.

Reply:
👍  When testing I'll double check that's not an issue.  Thanks.